js/ez-field-edit.js

```diff
@@ -0,0 +1,122 @@
+(function () {
+    /**
+     * `<ez-field-edit>` is supposed to wrap one or several HTML5 inputs. It tracks user input,
+     *  and if an input is invalid, sets its own `invalid` property to true. It will also add an
+     * `ez-invalid` class on the related label. This class will be removed once the input gets valid.
```

Comment:
the class has changed, `ez-invalid-input` and it now adds the class on the first common label and input ancestor.